website_sale_hide_price/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Website Sale Hide Price
+=======================
+
+This module allows to have hide product prices on the website store.
+
+Configuration
+=============
+
+#. Go to *Customers* and choose one.
+#. Go to *Sales and Purchases* tab.
+#. In *Sales* group set *Show prices on website* on or off so this custumer can
+   see them or not.
+
+The default value is `True`, so every partner website user can see the prices.
+
+Usage
+=====
+
+If no user is logged in the prices won't be shown. Prices will be shown if any
```

Comment:
This is also configuration information. Put a comma after `logged in`. Put here where to go to see that there are no prices.